event-based-simulators/main/simulators.json

```diff
@@ -22,6 +22,28 @@
                     "frequency": 20
                 }
             }
+        ],
+        "measurements": [
+            {
+                "type": "c8y_Temperature",
+                "series": "c8y_Temperature.T1",
+                "unit": "C",
+                "valueDistribution": "normal",
+                "mu": 20.0,
+                "sigma": 0.5,
+                "minimumFrequency": 60.0,
+                "maximumFrequency": 60.0
+            },
+            {
+                "type": "c8y_Temperature",
+                "series": "c8y_Temperature.T2",
+                "unit": "C",
+                "valueDistribution": "uniform",
+                "minimumValue": -20.0,
+                "maximumValue": 60.0,
+                "minimumFrequency": 60.0,
+                "maximumFrequency": 60.0
+            }
```

Comment:
I talked to @namnhatpham about this. Why do we need new profiles for it? I would think we could just have new Simulators that use measurements instead of events, but codewise they are both the same.